ansible_collections/arista/avd/examples/campus-fabric/documentation/devices/LEAF2A.md

```diff
@@ -72,16 +72,16 @@ interface Vlan10
    ip address 10.10.10.8/24
 ```
 
-### Name Servers
+### IP Name Servers
 
-#### Name Servers Summary
+#### IP Name Servers Summary
 
 | Name Server | Source VRF |
```

Comment:
The `Source VRF` is a bit misleading, since it is just the name server used when looking up something in xxx VRF. I suggest to just change to `VRF`. There is another configuration area around `ip_domain_lookup` which has source interface and VRF.